### PR TITLE
Categories Update Add Persian language

### PR DESCRIPTION
**Pull request by imanmoghimiq30**
Add Persian language 
اضافه کردن زبان فارسی لطفا در صورت یافتن مشکل پایم دهید یا اصلاح کنید

### CODE REVIEW

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
GitHub Watchdog Success http://angola.mapsme.mail.msk:8080/job/new_format_checker/
 No test results found.

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
GitHub Watchdog Success http://angola.mapsme.mail.msk:8080/job/new_format_checker/
 No test results found.

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
GitHub Watchdog Success http://angola.mapsme.mail.msk:8080/job/new_format_checker/
 No test results found.

**Comment by imanmoghimiq30** [comment]
please help! I no have idea what is wrong

**Comment by mapsmetest** [comment]
Format Checker

**Comment by mapsmetest** [comment]
Format Checker

File: data/categories.txt [hunk 1]
```diff
@@ -8896,6 +9159,3 @@ sponsored-thor
 en:Collect all the items and win prizes!
 ru:Соберите все и выиграйте призы!
 
-sponsored-halloween
```
Comment (mpimenov) [inline data/categories.txt]: Please revert the change for this category.

**Comment by mpimenov** [comment]
@imanmoghimiq30 There's nothing to worry, our bot is throwing tantrums.

Please sign the CLA: https://maps.me/cla/

**Comment by mpimenov** [comment]
NOTE. This change will break the build, so whoever merges it will have to fix indexer/categories_holder.cpp as soon as possible.

File: data/categories.txt [hunk 2]
```diff
@@ -8895,7 +9158,4 @@ zh-Hant:^山屋
 sponsored-thor
 en:Collect all the items and win prizes!
 ru:Соберите все и выиграйте призы!
-
-sponsored-halloween
-en:Let's check your luck. Trick or treat?
-ru:Проверим вашу удачу? Нажмите на «Trick» или «Treat»
+ -sponsored-halloween
```
Comment (mpimenov) [inline data/categories.txt]: Please, do not change the existing lines in this PR.
It currently shows "+266/-6", it should show "+265/-0"
